src/casus/saveCasus.js

```diff
@@ -0,0 +1,12 @@
+//@flow strict
+
+//TODO: get cookies saved on login
+//the cookies don't actually exist at the moment, so waiting on those getting pushed
+//import Cookies from 'universal-cookie';
+
+function saveCasus(): void {
+	console.log('saving...');
```

Comment:
I know this is trash and in general shouldn't be pushed, but I was in the middle of doing this...